### ISSUE TITLE
Improve CONTRIBUTING guide

### ISSUE THREAD

**Issue by gooshie** [FOSH-following-demand/FOSH-following-demand.github.io#20]
Remove **Style Guide** section about Python or change to something appropriate for this project?

**Comment by emdupre** [comment]
We could also clarify our description of issues, milestones, and project boards ! See #17 

One more idea: Right now we don't have material about working with Jekyll. Maybe that could go in place of the style guide section ?

**Comment by gooshie** [comment]
I agree @emdupre , this is the part of the project I'm eager to learn, and I'm thankful you are patiently guiding me through this process.
You had mentioned before that a Jekyll tutorial would be useful, so I was already thinking about creating it. I could put some of it on YouTube or downloadable PDF or webpage or all of the above. I just cannot start until the 15th of December. **[Here is a video](https://www.youtube.com/watch?v=McJasWE_alU)** I made recently to demonstrate Firefox's (most any browser's) development features. There is no sound. I'd like to have a better version of that video with sound as part of a tutorial on how to work through a Jekyll issue. If I make a tutorial for my YT channel, I'd like to show a series of tutorials covering the basics needed to go from blank slate to a simple blog site with a few "non-blog" pages. What do you think should be covered in a Jekyll tutorial?

**Comment by amchagas** [comment]
Hey! 
somehow this fell under the radar... I made some small changes to the CONTRIBUTING guide, adding a bit on Jekyll and on the issue labels.. suggestions and comments are welcome!
